docs/docs.trychroma.com/tsconfig.json

```diff
@@ -12,9 +16,17 @@
     "module": "esnext",
     "moduleResolution": "node",
     "resolveJsonModule": true,
-    "isolatedModules": true,
+    "isolatedModules": false,
```

Comment:
would love to why this has been changed

Reply:
<img width="1401" alt="CleanShot 2024-09-09 at 11 42 39@2x" src="https://github.com/user-attachments/assets/5749df55-def4-447f-a775-7fa6ff6ad314">

when it is on - it makes `yarn gen-js` very angry in our docs.